app/controllers/client2controller.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+class ClientsController extends \BaseController {
+
+	/**
+	 * Display a listing of the resource.
+	 * GET /clients
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{	
+		$pTitle		= "Clients";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$pTitle" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.144:9000/coding_rules#rule_key=php%3AS1481)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Clients" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://172.27.59.144:9000/coding_rules#rule_key=php%3AS1192)